bench/src/main/scala/parsley/bench/JsonBench.scala

```diff
@@ -0,0 +1,51 @@
+package parsley.bench
+
+import java.util.concurrent.TimeUnit
+import org.openjdk.jmh.annotations._
+import org.typelevel.jawn.ast._
+import parsley._, Combinator._, Parsley._
+import scala.io.Source
+
+/* Based on https://github.com/typelevel/jawn/blob/v1.0.0/benchmark/src/main/scala/jawn/JmhBenchmarks.scala */
+@State(Scope.Benchmark)
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+abstract class JmhBenchmarks(name: String) {
+  val text: String =
+    Source.fromResource(name, getClass.getClassLoader).getLines().mkString("\n")
+
+  def json: Parsley[JValue] = {
+    val jsontoks = LanguageDef("", "", "", false, NotRequired, NotRequired, NotRequired, NotRequired, Set.empty, Set.empty, true, Predicate(Char.isWhitespace))
```

Comment:
Any reason you didn't use `val jsontoks = LanguageDef.plain.copy(space = Predicate(Char.isWhitespace))`?

Reply:
Perhaps this is an even older version of the benchmark parser than the one on `brexit`...

Reply:
I exhumed this from a GitHub search and didn't find the head of the branch it lived on.  Yes, this is much nicer.